smarts/core/vehicle.py

```diff
@@ -265,7 +266,9 @@ def state(self) -> VehicleState:
             pose=self.pose,
             dimensions=self._chassis.dimensions,
             speed=self.speed,
+            # pytype: disable=attribute-error
             steering=self._chassis.steering,
+            # pytype: enable=attribute-error
```

Comment:
This means that the `Chassis` interface is not well defined. I could imagine steering  should be a property of the `Chassis` abstract class.

Reply:
`steering` is a property of the `Chassis` abstract class which I am not sure why pytype does not recognize. https://github.com/huawei-noah/SMARTS/blob/652aa23e71bd4e2732e2742140cfcd0ec082a7da/smarts/core/chassis.py#L134

Reply:
No, that didn't fix it. It is only `chassis.steering` that is not recognized properly for some reason.

Reply:
Is it because there's no return type specified?  (`Optional[float]`)